pkg/collect/cluster_resources.go

```diff
@@ -443,11 +439,17 @@ func crs(
 		}
 		apiResourceList, _ := apiResourceListObj.(*metav1.APIResourceList)
 		groupVersion := apiResourceList.GroupVersion
+
+		var namespace string
+		if len(namespaces) == 1 {
```

Comment:
I don't get this - can you explain? 
When this function gets more than one member in the namespaces, we won't use them at all - that seems bizarre.

Reply:
The context behind this implementation goes like this - 

We can either specify to collect the bundle from a single namespace (`-n <mynamespace>`) or from all namespaces. What happens in case of `SupportBundleCollector` is if we specify the namespace in the flag, the `namespaceNames` list contains only that namespace. Otherwise, all the namespaces are fetched and added to the list.

>When this function gets more than one member in the namespaces, we won't use them at all

From the above, we know that either it is a single namespace or all namespaces. That's why when we have the list greater than 1 element, we simply keep `namespace := ""` containing empty string. which means that all the custom resources will be fetched from all the namespaces. 
```
customResourcesResponse, err := client.RESTClient().Get().AbsPath("/apis/" + groupVersion).Namespace(namespace).Resource(customResourceName).DoRaw(ctx)
```

Reply:
Can you add this explanation as a comment above this code?